transformer_lens/loading_from_pretrained.py

```diff
@@ -557,6 +558,7 @@
     ],
     "mistralai/Mistral-7B-v0.1": ["mistral-7b"],
     "mistralai/Mistral-7B-Instruct-v0.1": ["mistral-7b-instruct"],
+    "mistralai/Mistral-7B-Instruct-v0.2": ["mistral-7b-instruct"],
```

Comment:
Wait, unless I miss something this means that this CHANGES which model `"mistral-7b-instruct"` points to. This seems unintended behavior -- can the alias be `"mistral-7b-instruct-v0.2"` instead?

Reply:
Ah yes, sorry for the oversight. I changed the alias to `mistral-7b-instruct-v0.2` :)